assets/src/dashboard/components/cardGallery/components.js

```diff
@@ -92,7 +92,6 @@ export const MiniCard = styled.div(
     cursor: pointer;
     border: 1px solid ${theme.colors.border.defaultNormal};
     border-radius: ${theme.borders.radius.small};
-    overflow: hidden;
```

Comment:
this was intentionally added to fix the border radius: 
![overflow hidden intentional](https://user-images.githubusercontent.com/10720454/111345132-388e3100-863a-11eb-9f0d-f26fba87cb41.gif)